### PR TITLE
Add cloud quickstart to nav menu and drop toc to make video fit

### PR DESCRIPTION
**Pull request by discdiver**
I don't think we intended to remove the cloud quickstart from the docs all together. The page still exists. This PR puts it in the nav under Cloud. It also makes the video image render better by removing the TOC for that page.

### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [x] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`, `docs`.
  

For documentation changes:

- [ ] This pull request includes redirect settings in `netlify.toml` for files that are removed or renamed

### CODE REVIEW

**Comment by billpalombi** [comment]
In the preview, I see the page under the "Cloud > Users" nav category, rather than one level up under "Cloud" directly.

**Comment by discdiver** [comment]
> In the preview, I see the page under the "Cloud > Users" nav category, rather than one level up under "Cloud" directly.

Ah, yes. Good catch. Moved under the correct index now.

**Review by billpalombi — APPROVED** [review]
lgtm, but consider just "Quickstart" instead of "Cloud Quickstart" since its now in the "Cloud" section.

File: mkdocs.yml [hunk 1]
```diff
@@ -164,6 +164,7 @@ nav:
                 - Automations: concepts/automations.md
     - Cloud:
           - cloud/index.md
+          - Cloud Quickstart: cloud/cloud-quickstart.md
```
Comment (discdiver) [inline mkdocs.yml:167]:
```suggestion
          - Quickstart: cloud/cloud-quickstart.md
```

**Comment by discdiver** [comment]
> lgtm, but consider just "Quickstart" instead of "Cloud Quickstart" since its now in the "Cloud" section.

Good idea. Changed.

**Pull request merged**